### PR TITLE
[Feature Fix] Profile settings reorder fix

### PR DESCRIPTION
**Pull request by leb2dg**
# Purpose

This resolves issue #1506 . The last and first position reordering for the employment and education tag in Profile Information was non-functional.

# Changes

The the default setting for tolerance of a sortable instance is intersect (elements must overlap more than 50%). That amount of overlap was not possible for the first and last items of the list. Set tolerance to pointer so that the overlap of elements is determined by where the mouse is located on the screen.
Reference: [jQuery UI API] (http://api.jqueryui.com/sortable/#option-tolerance)

# Side effects

The entries are reordered based off of any overlap of the pointer and another entry.

### CODE REVIEW

**Comment by sloria** [comment]
Nice work! :octopus:

File: website/static/vendor/knockout-sortable/knockout-sortable.js [hunk 1]
```diff
@@ -351,7 +351,8 @@
         },
         connectClass: ko.bindingHandlers.sortable.connectClass,
         options: {
-            helper: "clone"
+            helper: "clone",
+            tolerance: "pointer"
```
Comment (sloria) [inline website/static/vendor/knockout-sortable/knockout-sortable.js]: Ah, I just noticed that this is an edit to knockout-sortable. In general, we don't want to edit 3rd-party dependencies directly. Instead use configuration/extension when possible. @leb2dg Is this change necessary?

**Comment by leb2dg** [comment]
@sloria I think that changing the settings in the two templates is enough to fix the problem. Should I change knockout-sortable.js back to its original and create a pull request?

**Comment by sloria** [comment]
@leb2dg That would be great. Thanks!

**Comment by lbanner** [comment]
Great work @leb2dg !!
